docs/source/api/plugin/landing-pages.md

```diff
@@ -219,6 +233,22 @@ An object containing initial HTTP header values to populate in the Explorer on l
 <tr>
 <td>
 
+###### `embed`
+
+`boolean | ApolloServerPluginEmbeddedLandingPageProductionConfigOptions`
```

Comment:
this made the table hella ugly

<img width="1260" alt="image" src="https://user-images.githubusercontent.com/16724/169629479-44e2f1ee-aea5-42bb-be15-9368f0a48856.png">

do you know how to run the docs yourself btw? not sure why deploy preview isn't working.

Reply:
@StephenBarlow do you know the secret here? :)